livekit-agents/livekit/agents/telemetry/traces.py

```diff
@@ -231,6 +231,49 @@ def _to_proto_chat_item(item: ChatItem) -> agent_pb.agent_session.ChatContext.Ch
     return item_pb
 
 
+def _to_log_chat_item(item: ChatItem) -> map[str, any] | None:
+    if item.type == "message":
+        return {
+            "type": item.type,
+            "id": item.id,
+            "role": item.role,
+            "content": [content for content in item.content if isinstance(content, str)],
```

Comment:
should we include summaries of the non-text content? dropping this might remove useful context